docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3'
+services:
+  windbreaker:
+    build:
+      context: ./repos/windbreaker
+      dockerfile: Dockerfile_bundle
+    depends_on:
+      - postgres
+      - rabbitmq
+    environment:
+      SERVICE_ENVIRONMENT: docker
+  rss-watcher:
+    build:
+      context: ./repos/rss-watcher
+      dockerfile: Dockerfile_bundle
+    depends_on:
+      - postgres
+      - rabbitmq
+  postgres:
+    image: 'postgres:9.6.3-alpine'
+    ports:
+      - 5432
+  rabbitmq:
+    image: 'rabbitmq:3.6.10-alpine'
+    ports:
+      - 5672
```

Comment:
I originally wanted to use docker's ability to layer compose files, but right now there's a limitation on how build contexts are resolved that are causing issues with builds.

Reply:
Is this a known limitation that has an associated github issue, or just a general limitation that isn't expected to change?

Reply:
not a problem anymore